src/common/utils.ts

```diff
@@ -1,15 +1,16 @@
 // @ts-check
-import axios from "axios";
+import axios, { AxiosRequestConfig, AxiosRequestHeaders } from "axios";
 import toEmoji from "emoji-name-map";
 import wrap from "word-wrap";
-import { themes } from "../../themes/index.js";
+import { themes } from "../../themes/index";
+import type { ThemeEnum } from "../../themes/index";
 
 /**
  * @param {string} message
  * @param {string} secondaryMessage
  * @returns {string}
  */
-const renderError = (message, secondaryMessage = "") => {
+const renderError = (message: string, secondaryMessage = "") => {
```

Comment:
We set message to string but we didn't set secondaryMessage to string. In parameters we should be explicit about the types.